lib/xo.py

```diff
@@ -1,15 +1,21 @@
 import json
 import subprocess
 
-def xo_cli(action, args={}, check=True, simple_output=True):
+def xo_cli(action, args={}, check=True, simple_output=True, use_json=False):
+    run_array = ['xo-cli', action]
+    if use_json:
+        run_array += ['--json']
+    run_array += ["%s=%s" % (key, value) for key, value in args.items()]
     res = subprocess.run(
-        ['xo-cli', action] + ["%s=%s" % (key, value) for key, value in args.items()],
+        run_array,
         stdout=subprocess.PIPE,
         stderr=subprocess.STDOUT,
         check=check
     )
-    if simple_output:
+    if simple_output and not use_json:
         return res.stdout.decode().strip()
+    elif use_json:
+        return json.loads(res.stdout)
```

Comment:
To me, json.loads should be called on the decoded output, and not in the case when simple_output is False.

Untested:
```suggestion
    if simple_output:
        output = res.stdout.decode().strip()
        if use_json:
            return json.loads(output)
        return return output       
```